ACPs/196-precompile-secp256r1/README.md

```diff
@@ -0,0 +1,83 @@
+# ACP-196: Precompile for secp256r1 Curve Support
+
+| ACP | 196 |
+| :--- | :--- |
+| **Title** | Precompile for secp256r1 Curve Support |
+| **Author(s)** | [Santiago Cammi](https://github.com/scammi), [Arran Schlosberg](https://github.com/ARR4N)  |
+| **Status** | Proposed ([Discussion](https://github.com/avalanche-foundation/ACPs/discussions/XXX)) |
+| **Track** | Standards |
+
+## Abstract
+
+This proposal introduces a precompiled contract that performs signature verifications for the secp256r1 elliptic curve on Avalanche's C-Chain and Subnet-EVM chains. The precompile will be implemented at address `0x0000000000000000000000000000000000000100` and will enable native verification of P-256 signatures, significantly improving gas efficiency for biometric authentication systems, WebAuthn, and modern device-based signing mechanisms.
```

Comment:
```suggestion
This proposal introduces a precompiled contract that performs signature verifications for the secp256r1 elliptic curve on Avalanche's C-Chain. The precompile will be implemented at address `0x0200000000000000000000000000000000000006` and will enable native verification of P-256 signatures, significantly improving gas efficiency for biometric authentication systems, WebAuthn, and modern device-based signing mechanisms.
```

L1s (previously called subnets) are sovereign and can add their own precompiles as they wish, so this is just for C-Chain. We use different address ranges and I _think_ I've set the next one correctly, but we can check that later.